### ISSUE TITLE
Separate tool to collect liquidity per account per market

### ISSUE THREAD

**Issue by MarkoPaasila** [Codaone/DEXBot#416]
To further incentivize market making, exchanges need a tool to collect data. It should take a list of markets, then monitor them constantly. On each block, it should add the total USD value of the liquidity provided per account to a per-account-total. That means that time and amount matter. 

At request or constantly, a csv or similarly useful file should be updated, so the data can be further analyzed as a spreadsheet.

**Comment by MarkoPaasila** [comment]
As this would further the case of DEXBot - liquidity - I think it fits into the scope of the project.

**Comment by PermieBTS** [comment]
I like it
Let's revisit this in a month.

**Comment by cloud-8** [comment]
So this task is to provide sum total of amount of the asset in either side of a market? E.g. ZEPH/BTS it would provide total ZEPH (sum of all open order + balance) and total BTS (sum of all open orders + balance). Then provide a spreadsheet of every block? keep in mind every block is quite alot of data, most orders dont get filled for days in many markets, thus there will be 99.99% bloat in a spreadsheet.

**Comment by MarkoPaasila** [comment]
@cloud-8 no, the point would be to return a list of accounts and total amount of liquidity or volume they have provided each. 

something like:
$ monitor-liquidity.py btsusd
which would return a table with following columns:
account, liquidity_asset1, liquidity_asset2, volume_asset1, volume_asset2

I don't know if it should be _oneshot_, on a continuing basis, or something else. I'm not sure how much interest there would be for this. Maybe the exchanges could do this themselves if they need it.
